src/opendr/perception/activity_recognition/continual_transformer_encoder/continual_transformer_encoder_learner.py

```diff
@@ -0,0 +1,708 @@
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from functools import partial
+import json
+import torch
+import os
+import pickle
+from pathlib import Path
+from opendr.engine.learners import Learner
+from opendr.engine.helper.io import bump_version
+import onnxruntime as ort
+from collections import OrderedDict
+
+from opendr.engine.data import Timeseries, Vector
+from opendr.engine.datasets import Dataset
+from opendr.engine.target import Category
+
+from logging import getLogger
+from typing import Any, Union, Dict
+
+import pytorch_lightning as pl
+import continual as co
+from continual import onnx
+
+logger = getLogger(__name__)
+
+
+class TransformerEncoderWithPositionalEncoding(torch.nn.Module):
+    def __init__():
+        super().__init__()
+        pass
+
+    def forward():
+        pass
```

Comment:
Is this class actually used?

Reply:
No it's not. Not sure how I missed that one 😅 I've removed it in a follow-up commit. Thanks.